extensions/ql-vscode/.eslintrc.js

```diff
@@ -5,16 +5,23 @@ module.exports = {
     sourceType: "module",
     project: ["tsconfig.json", "./src/**/tsconfig.json", "./gulpfile.ts/tsconfig.json", "./scripts/tsconfig.json", "./.storybook/tsconfig.json"],
   },
-  plugins: ["@typescript-eslint"],
+  plugins: [
+    "github",
+    "@typescript-eslint"
+  ],
   env: {
     node: true,
     es6: true,
   },
   extends: [
     "eslint:recommended",
-    "plugin:@typescript-eslint/recommended",
+    "plugin:github/internal",
+    "plugin:github/react",
```

Comment:
I don't think we need to have the react rules on the top-level, only in the `src/view` directory.